docs/cron/export.sh

```diff
@@ -74,9 +74,9 @@ mysql --skip-column-names --batch tatoeba -e \
 # split user languages by language
 mysql --skip-column-names --batch tatoeba -e \
     "SELECT
-       COALESCE(ul.language_code, '\N'), 
-       COALESCE(ul.level, '\N'), 
-       COALESCE(u.username, '\N'), 
+       COALESCE(ul.language_code, '\\N'), 
+       COALESCE(ul.level, '\\N'), 
+       COALESCE(u.username, '\\N'), 
```

Comment:
We currently don't have a `username` that is `NULL` (and I'm pretty sure  we don't want to have one in the future) so I don't think you need to check for `NULL` here:
```
root@sloth:~# mariadb tatoeba -e "select count(*) from users where username is null;"
+----------+
| count(*) |
+----------+
|        0 |
+----------+
```